### DISCUSSION TITLE
Simple ideas to improve the UX

### DISCUSSION THREAD

**Discussion by al31c0** [matt8707/ha-fusion#433] [Ideas]
Hi, I just want to share some ideas to improve this amazing dashboard:

1. Add the possibility to lock the side-bar scroll
2. Add the possibility to adjust the zoom of the entire window (to set the correct view based on different devices)
3. Lock the top navigation bar when scrolling
4. Add the possibility to create horizontal scroll for horizontal view, even if the card are more than the horizontal space available
5. Add Chip object to object list (mainly for have fast and easy info (eg. number of lights that are on)
6. Modify Date object with just 1 row of text instead of 2 (e.g. Tuesday, 9 April)
7. Add possibility to add button to webcam expanded view

**Comment by LionCityGaming** [comment]
There is a way to lock the Sidebar scrolling, but it breaks the view on mobile: 

```
// select the <aside> element
const asideElement = document.querySelector('aside');

// add styles
if (asideElement) {
  asideElement.style.position = 'fixed';
  asideElement.style.height = '100vh';
  asideElement.style.overflowY = 'auto';
}
```
